flaskbb/utils/requirements.py

```diff
@@ -19,86 +19,95 @@
 
 
 class Has(Requirement):
+
     def __init__(self, permission):
         self.permission = permission
 
     def __repr__(self):
         return "<Has({!s})>".format(self.permission)
 
-    def fulfill(self, user, request):
```

Comment:
Trickiest part of the upgrade, honestly. There were reported issues with this not going well when this was made an optional argument in allows 0.5.0